docsource/modules150-160.rst

```diff
@@ -908,7 +908,7 @@ Module coverage 15.0 -> 16.0
 +-------------------------------------------------+----------------------+-------------------------------------------------+
 | |new| website_sale_loyalty                      |                      |                                                 |
 +-------------------------------------------------+----------------------+-------------------------------------------------+
-| |new| website_sale_loyalty_delivery             |                      |                                                 |
+| |new| website_sale_loyalty_delivery             | Nothing to do        |                                                 |
```

Comment:
```suggestion
| |new| website_sale_loyalty_delivery             | Done                 | Renamed from website_sale_coupon_delivery       |
```

Reply:
https://github.com/OCA/OpenUpgrade/blob/3061ff30d09754ac158c860538165bffdb3672f2/openupgrade_scripts/apriori.py#L14

Reply:
And you have to put the contrary one (`Renamed to...`) on `website_sale_coupon_delivery` del entry.

Reply:
Done!